server/apps/distributed-app/docs/modules/ROOT/pages/operate/webadmin.adoc

```diff
@@ -1206,6 +1206,51 @@ This task could be run safely online and can be scheduled on a recurring
 basis outside of peak traffic by an admin to ensure Cassandra message
 consistency.
 
+=== Deleting old messages of all users
+
+*Note:*
+Consider enabling the xref:configure/vault.adoc[Deleted Messages Vault]
+if you use this feature.
+
+Old messages tend to pile up in user INBOXes. An admin might want to delete
+these on behalf of the users, e.g. all messages older than 30 days:
+....
+curl -XDELETE http://ip:port/messages?olderThan=30d
+....
+
+link:#_endpoints_returning_a_task[More details about endpoints returning a task].
+
+The `olderThan` parameter should be expressed in the following format: `Nunit`.
+`N` should be strictly positive. `unit` could be either in the short form
+(`d`, `w`, `y` etc.), or in the long form (`days`, `weeks`, `months`, `years`).
+The default unit is `days`.
+
+Response codes:
+
+* 201: Success. Corresponding task id is returned.
+* 400: Error in the request. Details can be found in the reported error.
+
+To delete old mails from a different mailbox than INBOX, e.g. a mailbox
+named "Archived" :
+....
+curl -XDELETE http://ip:port/messages?mailbox=Archived&olderThan=30d
+....
+
+Since this is a somewhat expensive operation, the task is throttled to one user
+per second. You may speed it up via `usersPerSecond=10` for example. But keep
+in mind that a high rate might overwhelm your database or blob store.
+
+*Cassandra only:*
```

Comment:
```suggestion
*Scanning search only:*
```

Reply:
This is correct from a technical point of view, but will the average user understand its implications? Even I am not sure which James variants use "scanning search", I only know Cassandra works because I tested it. Could you tell me which other variants will work too?
Maybe "Scanning search only (e.g. Cassandra)" ?

Reply:
Then let me suggest:

```suggestion
*Scanning search only:* (unsupported for Lucene and ElasticSearch search implementations).
```

You can very well use cassandra in conjunction with ElasticSearch. That's the norm more than the exception.

Reply:
Right, that was our special setup. Makes sense this way.